test/Microsoft.ML.Tests/TimeSeries.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Api;
+using System;
+using System.Collections.Generic;
+using Xunit;
+using Microsoft.ML.Runtime.TimeSeriesProcessing;
+
+namespace Microsoft.ML.Tests
+{
+    public sealed class TimeSeries
+    {
+
+        public class Prediction
+        {
+            [VectorType(4)]
+            public double[] Change;
+        }
+
+        sealed class Data
+        {
+            public float Value;
+
+            public Data(float value)
+            {
+                Value = value;
+            }
+        }
+
+        [Fact]
+        public void ChangeDetection()
```

Comment:
The seasonality one does call into MKL

---
In reply to: [219570581](https://github.com/dotnet/machinelearning/pull/977#discussion_r219570581) [](ancestors = 219570581,219570371)

Reply:
The test coverage is adequate for now. There were no tests before. This is a port to unblock a team that is currently using TLC binaries and we want them to start using ML.NET. There will be many more iterations in this area that will add API and more tests. 

---
In reply to: [221325088](https://github.com/dotnet/machinelearning/pull/977#discussion_r221325088) [](ancestors = 221325088,219570673)

Reply:
I would disagree. 
We have TestDataPipeMore.cs with baseline tests, and we also have EntryPoint tests in other repo, so why not bring them?

---
In reply to: [221331846](https://github.com/dotnet/machinelearning/pull/977#discussion_r221331846) [](ancestors = 221331846,221325088,219570673)

Reply:
I didn't realize there were entrypoints test until later today but upon further investigation it seems we don't have datasets for some of those test, A4Benchmark-TS1.csv is missing and many of those tests are disabled for Linux because they don;t pass. Time Series won't be announced until 0.7 and legacy pipeline would be gone at that time. I'll port over whatever tests I can. 

---
In reply to: [221392605](https://github.com/dotnet/machinelearning/pull/977#discussion_r221392605) [](ancestors = 221392605,221331846,221325088,219570673)